downloads/index.md

```diff
@@ -304,12 +304,12 @@ The platforms currently supported by Julia are listed below. They are divided in
       <td> <font color="orange">Tier 2</font> </td>
     </tr>
     <tr>
-      <td> 7+ </td>
+      <td> 8+ </td>
       <td> i686 (32-bit) / x86-64 (64-bit) </td>
-      <td> <font color="crimson">Tier 3</font> </td>
+      <td> <font color="orange">Tier 2</font> </td>
```

Comment:
Is this change intended? I thought we do all of our Windows stuff with Windows 10 nowadays and don't check 8.

Reply:
We don't actively check it, but it is close enough to 10 that we expect it to keep working (for now).  We will deprecate it from the list whenever it is EOL from Microsoft.

Reply:
Windows 8 was EOL in January 2016. Windows 8.1 reached end of mainstream support (whatever it is) in January 2018, and will reach end of extended support in January 2013